lib/plugins/exchange/bitstamp.js

```diff
@@ -0,0 +1,21 @@
+const common = require('../common/ccxt')
+const _ = require('lodash/fp')
+const consts = require('./consts')
+const ORDER_TYPE = consts.ORDER_TYPES.MARKET
+const FIAT = common.FIAT['kraken']
+const CRYPTO = common.CRYPTO['kraken']
+
+const loadConfig = (account) => {
+  const mapper = {
+    'key': 'apiKey',
+    'clientId': 'uid'
+  }
+  const mapped = _.mapKeys(key => mapper[key] ? mapper[key] : key)(account)
+  return { ...mapped, timeout: 3000 }
+}
+
+const isConfigValid = ({ key, clientId, secret }) => key && secret && clientId
+
+const amountPrecision = () => 8

```

Comment:
I don't see why this would need to be a function, might as well export a constant.